features/integration/rekey.feature

```diff
@@ -6,18 +6,19 @@ Feature: Sending transactions
     And wallet information
 
   Scenario Outline: Sending transactions
-    Given default transaction with parameters <amt> "<note>"
+    When I generate a key using kmd for rekeying
+    Given default transaction with parameters <amt> "<note>" and rekeying key
```

Comment:
To avoid regressions, if this is a new rule, the test needs to be annotated such that it can be opted into.

Reply:
@winder Responding inline below - let me know if it helps.

> Why does this change and rekeying key?

* The intent is to minimize side effects stemming from rekeying. Add a new rule (_and rekeying key_) hints to use a different account (one specific to rekeying) rather than a base provided account.
* I'm open to renaming the rule.  I had tried to follow the same base pattern to show they're in the same neighborhood. 

> To avoid regressions

That's a good point.  Sorry about that, I overlooked it.  We'll either make a tag or wait until we propagate the change to the other SDKs.  Will discuss with @algochoi to make a call.

Reply:
After touching base with @algochoi, propose to _skip_ adding an annotation + merge the 4 downstream PRs concurrently.

Here's the rationale and open to discussing live:  
* I dislike breaking someone else's build, but unlike a new feature (e.g. new router), the PR fundamentally changes the repo.  
* Consequently, I think we want to force opting into the PR's approach.

Reply:
Updating here - All downstream SDKs are ready to be merged.

Reply:
There are multiple 3rd party SDKs which use these tests, so unfortunately it isn't enough to push all of our SDKs in unison.